draft-01/example.json

```diff
@@ -1,11 +1,12 @@
 {
     "$schema": "../draft-01/feedPage.schema.json",
-    "userId": "d448f1562c20dbafa42badd9f88560cd1adb2f177b30f0aa048cb243e55d37bd",
+    "_self": "sky://ed25519-d448f1562c20dbafa42badd9f88560cd1adb2f177b30f0aa048cb243e55d37bd/skyfeed-dev.hns/localhost/posts/page_1.json",
```

Comment:
Why drop the `userId`? As I understand is to be compatible with DAC standards but I am not sure I got the point.
Can we have something like `userId`, `contentRecordPath`, `skyApp` and `filePath` or a separate schema for that? @redsolver

Reply:
The `_self` field contains the `userId`, so it's redundant